packages/asset-service/src/generateAssetData/ethTokens/index.ts

```diff
@@ -37,7 +37,7 @@ export async function getTokens(): Promise<TokenAsset[]> {
   const network = NetworkTypes.MAINNET
   const contractType = ContractTypes.ERC20
 
-  const tokens = uniswapTokenData.tokens.map((token) => {
+  const tokens = uniswapTokenData.tokens.flatMap((token) => {
```

Comment:
```suggestion
  const tokens: TokenAsset[] = uniswapTokenData.tokens.flatMap((token) => {
```

to ensure this `flatMap` change doesn't have unintended consequences